bevy_lint/src/lints/restriction/missing_reflect.rs

```diff
@@ -221,34 +211,32 @@ struct TraitType {
 }
 
 impl TraitType {
-    fn from_local_crate<'tcx>(
-        tcx: TyCtxt<'tcx>,
-        trait_path: &[&str],
-    ) -> impl Iterator<Item = Self> + use<'tcx> {
+    fn from_local_crate<'tcx, 'a>(
```

Comment:
I'm not too pleased with this extra lifetime, but I don't know of a more elegant solution. The problem is that `cx` and `trait_path` are both references, but they don't have the `'tcx` lifetime, so we need to define a new one.